### ISSUE TITLE
Align dropdowns with WCAG rules (keyboard inputs)

### ISSUE THREAD

**Issue by ShrimpCryptid** [allen-cell-animated/timelapse-colorizer#408]
## Description

Dropdowns do not allow for arrow key selection, and also don't automatically show where the user last selected. Either revert to using just the Ant dropdown or implement the missing behaviors?

See https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/

## Expected Behavior

_What did you expect to happen instead?_

## Reproduction

_A minimal example that exhibits the behavior._

## Environment

_Any additional information about your environment. E.g. OS version, python version, browser etc._
